### PR TITLE
Upgrade Travis-CI env to Ubuntu 20.04

### PR DESCRIPTION
**Pull request by eecheng87**
`qemu-user` under Ubuntu 18.04 in Travis-CI doesn't support `qemu-riscv32`, as
a result, we upgrade it to Ubuntu 20.04 (focal)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,23 +4,25 @@ os:
...
   - gcc
 
 addons:
   apt:
     sources:
-      - sourceline: "deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ bionic-updates main universe"
+      - sourceline: 'ppa:chris-lea/redis-server'
     packages:
-      - qemu-user
+    - redis-tools
```
Comment (jserv) [inline .travis.yml:17]: Why are redis tools and server included?

Comment (eecheng87) [reply .travis.yml]: Removed

File: .travis.yml [hunk 2]
```diff
@@ -4,15 +4,15 @@ os:
...
+  - focal
 
 compiler:
   - gcc
 
 addons:
   apt:
     sources:
-      - sourceline: "deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ bionic-updates main universe"
+      - sourceline: 'ppa:chris-lea/redis-server'
```
Comment (jserv) [inline .travis.yml:15]: Is this change necessary?

Comment (eecheng87) [reply .travis.yml]: No, removed it.
